### ISSUE TITLE
Home network mixed architecture

### ISSUE THREAD

**Issue by mg64ve** [zim-desktop-wiki/zim-desktop-wiki#1360]
One simple question regarding Zim.
I am new to this protocol.
Suppose I have a home network with one raspberry pi 4, one Linux pc, one Windows laptop, one Android tablet and one Android phone.
How can I share from raspberry pi 4 some .Zim wiki files and access and change from all other devices?
Is that possible?

**Comment by Simplenuity** [comment]
Yes, it's possible.
For a similar scenario am using [Syncthing](https://syncthing.net). It is decentralized but one of my boxes is always running 24/7.
Otherwise I've successfully tested it with Nextcloud.
On Android devices I use a simple txt editor that comes with the file manager I use.

**Comment by mg64ve** [comment]
Nextcloud option seems to be more interesting.
Could you please tell me how is your setup?
What do you use in Linux and Windows? 
Is it possible to use wikiZim from Android and access Nextcloud Zim files?
Do you have a WebDav mapping?
Thanks @Simplenuity

**Comment by Simplenuity** [comment]
I've run it for a some time using a Nextcloud provider. That means on my Linux boxes (I don't use Windows) I've got the Nextcloud client installed to sync the relevant folders of the Zim notebooks.

On Android I used FolderSync to have the Zim notebooks available offline. There is no Zim version for Android but as the files are plain txt files they can be viewed/ edited with a simple txt editor of your choice. For all that there's no WebDav mapping necessary.

As mentioned I meanwhile prefer Syncthing for speed, flexibility, simplicity and reliability over Nextcloud. Latter I still use for syncing & backing up my contacts and from keeping them away from any Google account.

Hope that helps
